inbm/Changelog.md

```diff
@@ -10,7 +10,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
  - RTC 536601 - Added 'source' command to INBM. This command manages `/etc/apt/sources.list` and `/etc/apt/sources.list.d/*` and associated gpg keys on Ubuntu.
+- RTC 537769 - Add URI checks from trusted repo for Source Application gpg key URI 
```

Comment:
Provide a more detailed changelog entry for the added feature, explaining the security implications and usage changes. [medium]